test/tests/jobs/CreateJobTest.cpp

```diff
@@ -280,14 +279,6 @@ struct CreateJobTest : tpunit::TestFixture {
         tester->executeWaitVerifyContent(command, "402 Malformed retryAfter");
     }
 
-    void retryUnique() {
```

Comment:
Why remove it instead of updating it to show the new behavior?

Reply:
Lol so I knew if I did it one way you'd ask for the other so I just went with the easiest solution first. Will update.

Reply:
lol, not really true, I am always up for having more tests, not less 😄